packages/suite/src/actions/recovery/recoveryActions.ts

```diff
@@ -79,14 +81,23 @@ const recoverDevice = () => async (dispatch: Dispatch, getState: GetState) => {
     dispatch(setError(''));
     dispatch(setStatus('in-progress'));
 
-    const response = await TrezorConnect.recoveryDevice({
+    const params = {
         type: advancedRecovery ? 1 : 0,
         word_count: wordsCount,
         passphrase_protection: DEVICE.DEFAULT_PASSPHRASE_PROTECTION,
+        // @ts-ignore https://github.com/trezor/connect/pull/710
```

Comment:
Looks like this is going to make it in connect 8.1.19, the ignore can be removed once rebased with Szymon's update.

Reply:
https://www.npmjs.com/package/trezor-connect waiting for 8.1.19 to be releaed